content/doc/developer/security/csp.adoc

```diff
@@ -95,6 +99,24 @@ You can generally use https://github.com/jenkinsci/stapler/blob/master/docs/jell
 
 An example of this is https://github.com/jenkinsci/jenkins/pull/6849[jenkinsci/jenkins#6849].
 
+==== Inline `st:bind` tags
```

Comment:
Is this supposed to be level 2 as next section is?
```suggestion
=== Inline `st:bind` tags
```

Reply:
Subsection of "Inline script block", same as "General" above.